src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/InteropHelpers.cs

```diff
@@ -223,6 +223,36 @@ public static unsafe char AnsiCharToWideChar(byte nativeValue)
             return PInvokeMarshal.AnsiCharToWideChar(nativeValue);
         }
 
+        internal static unsafe char* StringToBstrBuffer(string str)
+        {
+            if (str == null)
```

Comment:
This can just call `Marshal.StringToBSTR`. (Or the marshaller can just call `Marshal.StringToBSTR` directly.)